rad-identities/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "rad-identities"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+
```

Comment:
It would benefit from a default bin target unless it's never intended to be ran stand-alone. Was this the idea all along for "builtins", to only work through `rad-exe`?

Reply:
It should be only run through `rad-exe` really, but can be made available as `rad-identities-dev`. However, I thought we don't have bin targets here because of the lock file (or lack thereof)?